Assets/JavaScript/booksList.js

```diff
@@ -0,0 +1,44 @@
+let booksListArr = [];
+
+const newBook = document.querySelector('.booksList');
+const retData = JSON.parse(localStorage.bookListStorage);
+booksListArr = retData;
+
+function displayBooks() {
+  newBook.innerHTML = booksListArr
+    .map(
+      (x) => `
+    <section class="booksList">
+    <p>${x.title}</p>
+    <p>${x.author}</p>
+    <button onclick="removeBook('${x.title}', '${x.author}')">remove</button>
+    <hr></hr>
+    <section>
+    `,
+    )
+    .join('');
+}
+
+window.addEventListener('load', () => {
+  displayBooks();
+});
+
+// eslint-disable-next-line no-unused-vars
+function addBook() {
+  const bookTitle = document.querySelector('.title').value;
+  const bookAuthor = document.querySelector('.author').value;
+  booksListArr.push({ title: bookTitle, author: bookAuthor });
+  const str = JSON.stringify(booksListArr);
+  localStorage.setItem('bookListStorage', str);
+  displayBooks();
+}
+
+// eslint-disable-next-line no-unused-vars
+function removeBook(title, author) {
+  booksListArr = booksListArr.filter(
+    (x) => x.title !== title || x.author !== author,
+  );
+  const str = JSON.stringify(booksListArr);
+  localStorage.setItem('bookListStorage', str);
+  displayBooks();
+}
```

Comment:
- [ ] Kindly remove the `Eslint` disabling comments. `Eslint` helps us to write clean, maintainable and readable codes and using these disabling comments would defeat this purpose 💻

Reply:
Hi @Gambit142 thanks for pointing this out,
The reason I disabled eslint for the functions is that they are not being mentioned inside the JavaScript file, and have only been mentioned in the HTML file,